packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2572,6 +2572,7 @@ class TextSelectionGestureDetectorBuilder {
     _dragStartSelection = renderEditable.selection;
     _dragStartScrollOffset = _scrollPosition;
     _dragStartViewportOffset = renderEditable.offset.pixels;
+    _dragBeganOnPreviousSelection ??= _positionOnSelection(details.globalPosition, _dragStartSelection);
```

Comment:
Do you still need the `??` even though this is now in the `Start` handler?

Reply:
Good point i'll remove it.